site/pages/our-work/case-study/sky/index.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import Social from '../../../../components/social';
+import styles from './style.css';
+
+import headerImage from './images/header.jpg';
+import cmsIllustrationImage from './images/cms-illustration.jpg';
+import authorQuoteImage from './images/quote-author.jpg';
+
+import ListBox from '../../../../components/list-box';
+import Quote from '../../../../components/quote';
+
+import WhatToReadNext from '../what-to-read-next';
+import ContactUs from '../../../../slices/contact-us-slice';
+import ContactBox from '../../../../components/contact-box';
+
+type CaseStudyFinancialTimesProps = {
+  contactUsURL: string,
+};
+
+const social = {
+  title: 'The proof is in the pudding',
+  description: 'Keeping up with the Financial Times',
+  metaImage: headerImage,
+  url: 'https://red-badger.com/our-work/case-study/financial-times',
+};
+
+const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps) =>
+  <div className={styles.caseStudy}>
+    <Social {...social} />
+    <div className={styles.header}>
+      <div className={styles.header__container}>
+        <div className={styles.header__imageContainer}>
+          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+        </div>
+      </div>
+    </div>
+    <div className={styles.content}>
+      <div className={styles.content__wrapper}>
+        <h1 className={styles.content__mainTitle}>
+          Helping Sky customers help themselves, with a refreshed customer service website
+        </h1>
+      </div>
+      <div className={styles.overview}>
+        <Quote
+          author={{
+            name: 'David Crawford',
+            title: 'Head of Software Engineering, Sky',
+            image: authorQuoteImage,
+          }}
+          text={
+            'Red Badger are recognised leaders in Agile/Lean methodologies so they could immediately start working and deliver results alongside our internal team'
+          }
+        />
+        <ListBox
+          title="Results"
+          items={[
+            {
+              label: 'Tripled the number of issues resolved through online self help journey',
+            },
+            {
+              label: 'Doubled the customers serviced by the advanced diagnostics capability',
+            },
+            {
+              label: 'Serviced half a million customers to date',
+            },
+          ]}
+          itemClassName={styles.listBox__item}
+          labelClassName={styles.listBox__label}
+        />
+      </div>
+      <div className={styles.content__wrapper}>
+        <h2 className={styles.content__title}>
+          <span className={styles.content__redTitle}>
+            {"Let's make things better."}
```

Comment:
Also, maybe the single quote character should be ’ instead as its more commonly used